packages/core/src/App/Components/Elements/NotificationMessage/notification-banner.jsx

```diff
@@ -1,10 +1,11 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import classNames from 'classnames';
 import { isMobile } from '@deriv/shared';
 import { Button, Icon, Text } from '@deriv/components';
 
-const NotificationBanner = ({ header, message, primary_btn, img_src, img_alt, onClose }) => (
-    <div className='notification-banner'>
+const NotificationBanner = ({ className, header, message, primary_btn, secondary_btn, img_src, img_alt, onClose }) => (
+    <div className={classNames('notification-banner', `notification-banner__${className}`)}>
```

Comment:
```
    <div 
      className={classNames('notification-banner', {
         [`notification-banner__${className}`]: className
      })}
    >
```
Can you add this condition to check if the className is defined?

Reply:
Thanks for the suggestion 👍🏻  I added the condition.